### ISSUE TITLE
CoreData + Ensembles migration issue

### ISSUE THREAD

**Issue by sacred0x01** [drewmccormack/ensembles#291]
...

**Comment by drewmccormack** [comment]
With Ensembles, you must add an explicit model version when you change anything. Did you do that? 

You can still use lightweight migration, but the old version and the new version must be in the model. You can’t use the really new system where it just figures out the old model.

Kind regards,
Drew

**Comment by sacred0x01** [comment]
Hi, @drewmccormack! Thanks for your answer.
 
We did all the things, but issue still was on our side. See ![screenshot](https://i.ibb.co/LJN552R/Screen-Shot-2020-07-30-at-23-06-51.png) 
We were trying to use transformable type to store array without data wrapper, but received error about NSData archiving and switched to Binary Data type, but xcode decided not to remove previous fields. After hours of debugging your lib i found an issue with json importer and wrong type packing and fix it inside xcode model file.

Also, is it possible to use Transformable somehow with your lib? Because all array-like structure right now we have packed as Data.
